src/main/java/Deadline.java

```diff
@@ -0,0 +1,22 @@
+public class Deadline extends Task{
+
+    protected String by;
```

Comment:
Hi, I think you can consider making this a private variable! Same goes for the variables in other classes as well!